docs/sql-ref-syntax-qry-select.md

```diff
@@ -164,6 +164,11 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { [ named_expression | regex_column_na
      )
      ```
 
+* **TRANSFORM**
+
+     Specifies a hive-style transform (`SELECT TRANSFORM`) query specification to transform
```

Comment:
`(SELECT TRANSFORM)` looks redundant as this is already shown as the SQL syntax.

Reply:
Done